appveyor.yml

```diff
@@ -0,0 +1,60 @@
+environment:
+  matrix:
+  - ARCH: "i686"
+    CCPATH: 'C:\mingw-w64\i686-6.3.0-posix-dwarf-rt_v5-rev1\mingw32\bin'
```

Comment:
okay, well can you suggest which one I do use? The cygwin one didn't seem to work either.